README.md

```diff
@@ -1 +1,53 @@
-# Awesome_Books
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesom Books
+
+> This is a website that allows users to add/remove books from a list. This project is build using JavaScript objects and arrays.
+
+
+## Built With
+
+- HTML
+- CSS 
+- Javascript
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone repository in your local machine 
+- cd Portfolio
+- open `index.html` in your browser.
+- You can use live server.
+
+## Live Demo
+
+## Author
+
+👤 **Sajid Munawar**
+
+- GitHub: [@githubhandle](https://github.com/sajid-munawar)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/sajid-munawar-41ba26180/)
+
+ 👤 **Zelalem Fissha**
+
+- GitHub: [@githubhandle](https://github.com/zelalem1222)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the 
+
+## Show your support
+
+Give a ⭐️ to if you like this project!
+
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [OPTIONANL] Please update your Readme, currently some headings are missing such as  

``` ##  License ``` section with the MIT license at the end of your README file to make it more professional. I would also recommend you add this. 😄 

- please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.